regression/README.md

```diff
@@ -0,0 +1,58 @@
+# Welcome to the TDS Regression Test Suite Application
```

Comment:
We should have a similar README.md in the api regression project explaining how to run the tests in Maven and IntelliJ.  We should specify the property keys that need to be in the file.